cantools/database/can/formats/dbc.py

```diff
@@ -558,6 +572,13 @@ def _dump_senders(database):
 def _dump_comments(database):
     cm = []
 
+    if database.dbc is not None:
+        if database.dbc.comment is not None:
+            cm.append(
+                'CM_ "{comment}";'.format(
+                    #comment=database.dbc.comment.replace('"', '\\"')))
+                    comment=database.dbc.comment) )
```

Comment:
Remove commented code.

Reply:
1. Tests: OK, I will add more tests.
2. I do not have specific reason between DbcSpecifics class vs Bus class. It is tricky to choose
I tried to duplicate comments and see how DBC viewer tool handles like
CM_ "Special Release";
CM_ "Special Release2";
tool combines them like: CM_ "Special ReleaseSpecial Release2";
& Bus class handles only 3 attributes, if  there is no Bus name and baudrate attribute in dbc file. comment  is only attribute left 

DbcSpecifics class handles attributes dictionary so looks more appropriate for me personally

3.  I will remove commented code

Reply:
Use the comment attribute in the Bus class. Do not add anything to the DbcSpecifiers class. The reason is that a comment seems generic over multiple database formats, and therefore belongs to the Bus class.

Reply:
It looks like only the last found CM_ for signals and messages are stored. I suggest using the same logic here. That is, not concatenating comments.